src/inference/src/ie_core.cpp

```diff
@@ -243,6 +243,8 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
             std::shared_ptr<ie::ICacheManager> _cacheManager;
         };
 
+        static bool flag_allow_auto_batching;
```

Comment:
Why set it as a static variable?

Reply:
no static variables in code

Reply:
> Why set it as a static variable?

Static variable is common to all objects, for multiple objects, static data members are only stored in one place for all objects to share. I found in this case, maybe we don't need to use a static variable. Now I update it.

Reply:
> no static variables in code

Update it in the new commit